### PR TITLE
feat: Add Olm tests for openshift ci and minikube

### PR DESCRIPTION
**Pull request by flacatus**
### What does this PR do?
This PR add additional tests for minikube and Openshift(will be 4.5 4.6 4.7) to test OLM installer.
I add all tests to deploy chectl in namespace eclipse-che
Refference issue: https://github.com/eclipse/che/issues/18293

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by flacatus** [comment]
@tolusha 
WDYT about renaming gh actions jobs like:
E2E Minikube / OLM installer
E2E Minikube / Helm installer
E2E Minikube / Operator installer
E2E Minishift / Operator installer

**Comment by tolusha** [comment]
I don't mind

File: test/e2e/e2e.test.ts [hunk 1]
```diff
@@ -166,7 +178,7 @@ describe('Workspace creation, list, start, inject, delete. Support stop and dele
     it('Testing workspace:inject command', async () => {
       console.log('>>> Testing workspace:inject command')
 
-      const { exitCode, stdout, stderr } = await execa(binChectl, ['workspace:inject', '--kubeconfig'], { timeout: 60000, shell: true })
+      const { exitCode, stdout, stderr } = await execa(binChectl, ['workspace:inject', '--kubeconfig', `-n ${NAMESPACE}`], { timeout: 30000, shell: true })
```
Comment (tolusha) [inline test/e2e/e2e.test.ts:181]: 60000 is better here

**Comment by flacatus** [comment]
New tests are created here https://github.com/openshift/release/pull/13771. Pending to merge this PR to have those tests green in OCP ci

**Review by tolusha — APPROVED** [review]

**Pull request merged**